File: src/EducationSystem/EducationSystem.Models/TestExecution.cs
using System.Collections.Generic;
using EducationSystem.Models.Datas;
using EducationSystem.Models.Rest;

namespace EducationSystem.Models
{
    public class TestExecution
    {
        public TestData TestData { get; set; }

        public List<Question> Questions { get; set; }

        public TestExecution(TestData testData, List<Question> questions)
        {
            TestData = testData;
            Questions = questions;
        }
    }
}

File: src/EducationSystem/EducationSystem.WebApp/App/src/models/TestExecution.ts
import Model from './Model'
import {TestType} from '../common/enums'
import Theme from './Theme'
import Discipline from './Discipline'
import TestData from './TestData'
import Answer from './Answer'
import Question from './Question'

export default class TestExecution extends Model {
  public Questions: Array<Question> = []
  public TestData?: TestData
}